app/services/xccdf_tailoring_file.rb

```diff
@@ -18,15 +18,41 @@ def initialize(profile:, rule_ref_ids: [], set_values: {})
 
   def builder
     @builder ||= create_builder do |xml|
-      xml[XCCDF].benchmark(id: @profile.benchmark.ref_id)
+      benchmark_builder(xml)
```

Comment:
btw, funnily enough, leaving off the benchmark completely is valid for the oscap eval run; I feel like we'll want to know which benchmark a tailoring file was generated from, though, so I'd rather have a kind of weird href value than no benchmark info at all.